docs/docs-beta/docs/guides/etl/ingesting-data.md

```diff
@@ -1,6 +0,0 @@
----
```

Comment:
Not sure where else to put this, but you'll also need to rename the `docs/guides/etl.md` file to `docs/guides/ingestion-and-transformation.md`. It's why the deploy preview is currently busted + complaining about a missing sidebar ID.